src/java.base/unix/classes/sun/nio/fs/UnixFileSystem.java

```diff
@@ -261,6 +261,9 @@ public final Path getPath(String first, String... more) {
         if (more.length == 0) {
             path = first;
         } else {
+            if (first == null) {
```

Comment:
`Objects.requireNonNull(first)` does this in fewer lines and a bit more explicitly.

Should it be checking `first` always, not just in the case `more.length != 0`?
(It will throw NPE, but not until `UnixPath.normalizeAndCheck(path))`.

Reply:
Good point on the Objects call. I checked the more.length == 0 case and let if fall through to the UnixPath method. It might be more straightforward to always check however.